.travis.yml

```diff
@@ -9,6 +10,10 @@ matrix:
       env: TOXENV=py35
     - python: 3.6
       env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8-dev
```

Comment:
Could this be updated to just `3.8` now, instead of `3.8-dev` ?

Reply:
Indeed :) changed